docs/source/installation.rst

```diff
@@ -1,21 +1,32 @@
 Installation
 ============
 
-PyTrack can be most easily installed via
+PyTrack can be most easily installed via pip:
 
-.. code-block::
+.. code-block:: bash
 
-    pip install py-track
+    pip3 install py-track
 
 If you want to install PyTrack from source you can use
 
-.. code-block::
+.. code-block:: bash
 
     git clone https://github.com/zincware/py-track.git
-    pip install .
+    cd py-track
+    pip3 install . --user
```

Comment:
On Windows I had issues with the `--user` and I would not recommend using it as a default.

Reply:
Okay